plans/feat-895-frontmatter-pr-b-server-wiki-bar.md

```
@@ -0,0 +1,116 @@
+    editor: meta.editor,
+  });
+  const finalContent = serializeWithFrontmatter(merged, parseFrontmatter(content).body);
+  await writeFileAtomic(absPath, finalContent, { uniqueTmp: true });
+  ```
+- `now` injection を opts に追加 (テスト可制)
+- caller が body を渡す前提だが、frontmatter を含む raw を渡してきても parse して body を取り出して merge
+
+### wiki/View.vue metadata bar
+
+- 上部に 1 行 bar:
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)